### PR TITLE
Assets: update logo bg/shape description, add assets app

### PR DESCRIPTION
**Pull request by catenocrypt**

### CODE REVIEW

**Comment by Nickk392** [comment]
Can I please have help updating our logo on trust wallet? image

VICEx 
Contract: 0xdfd5bb6ce8716822b2c9b255560194ca3509bebc

Thank you!

File: assets/add_new_asset.md [hunk 1]
```diff
@@ -180,10 +180,13 @@ Smart contract address update procedure:
...
 - File location: must be placed in the correct folder and subfolder within the [folder structure](#repository-structure).
-- File extension: `png`. Uppercase `PNG` is considered invalid.
-- File name：`logo.png`, all lowercase.
-- Dimension: `256 x 256 pixels` or `512 x 512 pixels`.
-- Background: Avoid transparency, set background as your brand color or white by default otherwise your token image wont look good on white/dark theme design.
+- File name：`logo.png`, all lowercase.  Extension: `png` (uppercase `PNG` is considered invalid).
+- Dimension: recommended `256 x 256 pixels`, maximum `512 x 512 pixels`, aspect ratio should be 1:1.
+- Edges and background:
+- Logos are displayed cropped to a circular mask, a circle fitting in the square shape.  Thus the corners of the logo image will not be visible.  The logo should fit in the circle, but also fill it as much as it possible, i.e. there should not be unused spaces on the sides/top/bottom.
+- Logos should look OK with white/lighgray background as well as on black/darkgray background (night mode).  For dark themed logos use white contour lines to make sure they stand out on dark background as well.
```
Comment (vorotech) [inline assets/add_new_asset.md:187]:
```suggestion
- Logos should look OK with white/lightgray background as well as on black/darkgray background (night mode).  For dark themed logos use white contour lines to make sure they stand out on dark background as well.
```

File: assets/add_new_asset.md [hunk 2]
```diff
@@ -180,10 +180,13 @@ Smart contract address update procedure:
 
 ### Image Requirements
 - File location: must be placed in the correct folder and subfolder within the [folder structure](#repository-structure).
-- File extension: `png`. Uppercase `PNG` is considered invalid.
-- File name：`logo.png`, all lowercase.
-- Dimension: `256 x 256 pixels` or `512 x 512 pixels`.
-- Background: Avoid transparency, set background as your brand color or white by default otherwise your token image wont look good on white/dark theme design.
+- File name：`logo.png`, all lowercase.  Extension: `png` (uppercase `PNG` is considered invalid).
+- Dimension: recommended `256 x 256 pixels`, maximum `512 x 512 pixels`, aspect ratio should be 1:1.
+- Edges and background:
```
Comment (vikmeup) [inline assets/add_new_asset.md:185]: This part does not look well

Screen Shot 2021-09-16 at 6 17 36 AM

File: assets/add_new_asset.md [hunk 3]
```diff
@@ -283,6 +285,22 @@ To do changes in the assets repository, you need to create a personal copy calle
 Once the changes are prepared inside the fork, you need to create a _pull request_ to the main repository.
 Upon review the maintainers will accept your pull request, and the changes will be incorporated.
 
+#### Adding files using Assets App
```
Comment (vikmeup) [inline assets/add_new_asset.md:288]: can you delete `Adding files using command-line` devs usually would know how to do it, and the rest can use assets.trustwallet.com

File: assets/add_new_asset.md [hunk 4]
```diff
@@ -283,6 +285,22 @@ To do changes in the assets repository, you need to create a personal copy calle
...
+
+A new token can be submitted using the Assets application.
+See also: https://community.trustwallet.com/t/how-to-submit-a-token-logo-using-assets-app/82957
+
+1. Open the assets web app:   [https://assets.trustwallet.com]( https://assets.trustwallet.com)
+2. Press the Log in with Github button.  If not yet logged in (in the current browser session), you need to log in to Github
+3. First time you will need to authorize the Assets app to access your Github account.
+4. The application should be now loaded
+5. Upload the logo file.
+6. Fill the token contract.  For some tokens (Erc20) some fields are auto-filled (symbol, decimals, etc.).
```
Comment (vikmeup) [inline assets/add_new_asset.md:298]:
```suggestion
6. Fill the token contract.  For some tokens (ERC20) some fields are auto-filled (symbol, decimals, etc.).
```

File: assets/add_new_asset.md [hunk 5]
```diff
@@ -283,6 +285,22 @@ To do changes in the assets repository, you need to create a personal copy calle
...
 
+#### Adding files using Assets App
+
+A new token can be submitted using the Assets application.
+See also: https://community.trustwallet.com/t/how-to-submit-a-token-logo-using-assets-app/82957
+
+1. Open the assets web app:   [https://assets.trustwallet.com]( https://assets.trustwallet.com)
+2. Press the Log in with Github button.  If not yet logged in (in the current browser session), you need to log in to Github
+3. First time you will need to authorize the Assets app to access your Github account.
+4. The application should be now loaded
```
Comment (vikmeup) [inline assets/add_new_asset.md:296]:
```suggestion
```

remove this step

**Review by vikmeup — APPROVED** [review]

**Pull request merged**